activities/philanthropic-fundraising/index.md

```diff
@@ -0,0 +1,14 @@
+---
+type: Index
+---
+
+# Philanthropic fundraising
+
+Our model is built on raising philanthropic funds. Gifts are tax-deductible. 
```

Comment:
This currently sounds like we plan to only rely on philanthropic gifts in the future. I don't think that's our official position.
```suggestion
Our [financial model](https://about.publiccode.net/organization/financial-model.html) includes raising philanthropic funds. Gifts are tax-deductible in both the Netherlands and the US.
```